scripts/changelog/CHANGELOG_MASTER

```diff
@@ -1,3 +1,10 @@
+1.38.0-1
+Sharanya Devaraj <sharanyd@amazon.com>
+Thu, 19 Mar 2020 13:42:00 PST
```

Comment:
when manually merging back to dev this should be changed to `2020-03-12T13:42:00-08:00`